kubernetes/launch-job.sh

```diff
@@ -201,8 +198,10 @@ ctl job run --container \
     "$SHUFFLE_AND_EXPORT_CMD" \
     "$CURRICULUM_CMD" \
     --high-priority \
+    --restart-on-failure \
+    --memory 72Gi 16Gi 72Gi 96Gi 4Gi \
```

Comment:
rationale: I looked at a cp505 warmstart run and saw the following numbers from `cat /sys/fs/cgroup/memory/memory.usage_in_bytes`:
* victimplay & train using 55Gi memory
* evaluate_loop using 9Gi memory
* shuffler using 64Gi memory
* curriculum using 0.4Gi memory